cmd/server/flags.go

```diff
@@ -36,6 +36,24 @@ var flags = append([]cli.Flag{
 		Name:    "log-xorm-sql",
 		Usage:   "enable xorm sql command logging",
 	},
+	&cli.IntFlag{
+		Sources: cli.EnvVars("WOODPECKER_XORM_MAX_CONNECTIONS"),
```

Comment:
Should get expert prefix and description hint.

Reply:
well I dont thing so ... if you config more connections for your db you can increase it here ...

and that's just an normal admin task

Reply:
@qwerty287 @zc-devs done